python/nano/src/bigdl/nano/quantization/neural_compressor/quantization.py

```diff
@@ -60,44 +64,63 @@ def __init__(self,
         :param inputs:      For tensorflow to specify names of inputs. e.g. inputs=['img',]
         :param outputs:     For tensorflow to specify names of outputs. e.g. outputs=['logits',]
         """
-        if conf:
-            qconf = Quantization_Conf(conf)
-        else:
-            qconf = Quantization_Conf('')
-            cfg = qconf.usr_cfg
-            # Override default config
-            cfg.model.framework = framework
-            cfg.quantization.approach = approach
-            cfg.tuning.strategy.name = tuning_strategy
-            if accuracy_criterion:
-                cfg.tuning.accuracy_criterion = accuracy_criterion
-            cfg.tuning.exit_policy.timeout = timeout
-            cfg.tuning.exit_policy.max_trials = max_trials
-            cfg.model.inputs = inputs
-            cfg.model.outputs = outputs
+        qconf = Quantization_Conf(conf)
+        cfg = qconf.usr_cfg
+        # Override default config
+        cfg.model.framework = framework
+        cfg.quantization.approach = approach
+        cfg.tuning.strategy.name = tuning_strategy
+        if accuracy_criterion:
+            cfg.tuning.accuracy_criterion = accuracy_criterion
+        cfg.tuning.exit_policy.timeout = timeout
+        cfg.tuning.exit_policy.max_trials = max_trials
+        cfg.model.inputs = inputs
+        cfg.model.outputs = outputs
         super().__init__(qconf)
 
+    def post_training_quantize(self, model, calib_dataloader=None, val_dataloader=None,
+                               metric=None):
+        self.check(calib_dataloader, val_dataloader, metric)
+        self.model = model
+        if calib_dataloader:
+            self.calib_dataloader = calib_dataloader
+        if val_dataloader:
+            self.eval_dataloader = val_dataloader
+            if metric:
+                framework = self.cfg.model.framework
+                if 'pytorch' in framework:
+                    framework_metric = METRICS['pytorch']
+                else:
+                    framework_metric = METRICS[framework]
 
-class TorchMetricForINC:
-    def __init__(self, metric):
-        self.metric = metric
-        self.pred_list = []
-        self.label_list = []
+                class MyMetric(framework_metric):
```

Comment:
`MyMetric` looks weird. Maybe briefly document why do you need this class here?

Reply:
Added document.